### ISSUE TITLE
Support for use as a javascript module

### ISSUE THREAD

**Issue by ldraw-org** [LasseD/buildinginstructions.js#60]
Maybe it's my lack of technical understanding and this is already possible but it would be helpful if I could use buildinginstructions.js files as a javascript modules with an "import" command.

**Comment by LasseD** [comment]
Nope. You are right. three.js has also moved in that direction now, so it is something that should be done.
It will also allow for pruning of three.js, so that the JS files can become smaller (and faster).
